### DISCUSSION TITLE
Deprecate stale repositories

### DISCUSSION THREAD

**Discussion by israel-hdez** [kiali/kiali#4357] [Governance]
I wonder if we should deprecate and archive the following repositories:

* ~~https://github.com/kiali/kiali-client-python~~ still used
* https://github.com/kiali/kiali-performance-testing
* https://github.com/kiali/kiali-design
* https://github.com/kiali/kiali-katacoda-scenarios

For all of them, last update was more than a year ago. Some have been stale for more than two years.

**Comment by lucasponce** [comment]
+1 to revisit and archive the repos that are not used.

I have some doubts about the kiali-client-python or kiali-performance-testing, @FilipB: do you remember if those are used in some integration test ?

**Comment by FilipB** [comment]
The kiali-performance-testing can be archived, it's not used anywhere.
The kiali-client-python is a rest client used by kiali ui automation suite and should not be archived.

**Reply by israel-hdez** [reply]
OK, I guess the other three can be archived, as I don't think we have plans to use them again.
Any other thoughts, @kiali/maintainers ?

**Reply by jshaughn** [reply]
I see no blockers.  I suggest starting a timer on this one.

**Reply by israel-hdez** [reply]
OK. I'll start a timer now @kiali/maintainers .
I'll wait until Oct 5th. If I see no push backs I'll archive the three repositories mentioned in the main comment.

**Comment by israel-hdez** [comment]
The mentioned repositories were archived. I'll lock this discussion.
